pyproject.toml

```diff
@@ -170,3 +170,48 @@ extend-ignore = [
 
 [tool.ruff.lint.flake8-builtins]
 builtins-ignorelist = ["id"]
+
+[tool.bumpversion]
+
+current_version = "4.11.0dev"
+commit = false
+tag = false
+parse = "(?P<major>\\d+)\\.(?P<minor>\\d+)\\.(?P<patch>\\d+)((?P<release>[a-z]+))?((?P<build>\\d+))?"
+serialize = [
+  "{major}.{minor}.{patch}{release}{build}",
+  "{major}.{minor}.{patch}{release}",
+  "{major}.{minor}.{patch}",
+]
+
+[tool.bumpversion.parts.release]
+
+optional_value = "stable"
+first_value = "dev"
+values = [
+  "dev",
```

Comment:
if you plan on backporting this, can we remove the `dev` suffix for stable branches (4.9 and 4.10), please?